File: Examples/Conversations/Obj-C/ConversationsExample/MyQuestionTableViewCell.m
//
//  MyQuestionTableViewCell.m
//  ConversationsExample
//
//  Copyright © 2016 Bazaarvoice. All rights reserved.
//

#import "MyQuestionTableViewCell.h"

@interface MyQuestionTableViewCell ()

@property(weak, nonatomic) IBOutlet UILabel *questionSummary;
@property(weak, nonatomic) IBOutlet UILabel *questionDetails;

@end

@implementation MyQuestionTableViewCell

- (void)setQuestion:(BVQuestion *)question {

  super.question = question;
  self.questionSummary.text =
      [NSString stringWithFormat:@"%@ (%lu Answers)", question.questionSummary,
                                 (unsigned long)question.includedAnswers.count];
  self.questionDetails.text = question.questionDetails;
}

@end

File: Examples/Conversations/Swift/ConversationsExample/MyQuestionTableViewCell.swift
//
//  MyQuestionTableViewCell.swift
//  Conversations
//
//  Copyright © 2016 Bazaarvoice. All rights reserved.
//

import UIKit
import BVSDK

class MyQuestionTableViewCell: BVQuestionTableViewCell {
  @IBOutlet weak var questionSummary : UILabel!
  @IBOutlet weak var questionDetails : UILabel!
  
  override var question: BVQuestion? {
    didSet {
      questionSummary.text = "\(question!.questionSummary!)  (\(question!.answers.count) Answers)"
      questionDetails.text = question!.questionDetails
    }
  }
}
